include/dlaf/communication/broadcast_panel.h

```diff
@@ -173,7 +173,7 @@ void broadcast(const comm::Executor& ex, comm::IndexT_MPI rank_root,
   const auto owner = dist.template rankGlobalTile<coordT>(last_tile);
   const auto range = dist.rankIndex().get(coordT) == owner
                          ? common::iterate_range2d(*panelT.iteratorLocal().begin(),
-                                                   LocalTileIndex(coordT, panelT.rangeEndLocal() - 1, 1))
+                                                   *std::prev(panelT.iteratorLocal().end()))
```

Comment:
No it doesn't work:
`*std::prev(panelT.iteratorLocal().end())` gives `LocalTileIndex(coordT, panelT.rangeEndLocal() - 1, 0)`
which results in an empty range.

Reply:
You're right! Because the range are defined with the first index outside, not the last one in the range. My error.

The other option would have been to use the overload that accepts as second parameter the size, by specifing the distance between the begin and std::prev(end), but it would require more "work" because the difference of two iterators is a linear index, while the iterate_range2d function requires a 2D size.

I would go for keeping it simple for the moment and just revert (again!) this last change.